crates/supervisor/core/src/lib.rs

```diff
@@ -7,8 +7,8 @@ pub use supervisor::{Supervisor, SupervisorError, SupervisorService};
 mod rpc;
 pub use rpc::SupervisorRpc;
 
-mod syncnode;
-pub use syncnode::{
-    AuthenticationError, ManagedNode, ManagedNodeConfig, ManagedNodeError, NodeEvent,
-    SubscriptionError,
-};
+/// Contains the managed node logic, which handles interactions with the op-node.
```

Comment:
better put module docs at start of the specific module with module doc syntax `//!`